jetty-util/src/main/java/org/eclipse/jetty/util/Pool.java

```diff
@@ -461,7 +461,13 @@ public void close()
         // iterate the copy and close its entries
         for (Entry entry : copy)
         {
-            if (entry.tryRemove() && entry.pooled instanceof Closeable)
+            boolean removed = entry.tryRemove();
+            if (!removed)
+            {
+                if (LOGGER.isDebugEnabled())
+                    LOGGER.debug("Pooled object still in use: {}", entry);
+            }
+            if (removed && entry.pooled instanceof Closeable)
```

Comment:
Can you flip the `if/else` for readability?

```java
boolean removed = entry.tryRemove();
if (removed) {
  if (instanceof Closeable) 
    ...
} else {
  if (LOGGER.isDebugEnabled())
     ...
}

Reply:
Done.